packages/xo-server-usage-report/src/index.js

```diff
@@ -122,6 +127,16 @@ Handlebars.registerHelper('shortUUID', uuid => {
   }
 })
 
+Handlebars.registerHelper(
+  'normaliseValue',
+  value => (isFinite(value) ? round(value, 2) : '-')
+)
+
+Handlebars.registerHelper(
+  'normaliseEvolution',
+  value => (isFinite(+value) ? `(${value > 0 ? '+' : ''}${value} %)` : '')
```

Comment:
Don't you want a `-` for negative values?

Reply:
The negative values have already `-`.